packages/vscode-extension/package.json

```diff
@@ -778,7 +778,7 @@
             ]
           },
           "args": {
-            "oneOf": [
+            "anyOf": [
```

Comment:
What's the effect after changed to `anyOf`? Will the warning still exist?

Reply:
The warning will disappear. It allow args to match multiple schema in the array.